src/views/Login.tsx

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 import LoginButton from '../components/LoginButton';
+import './Login.scss';
 
 const Login: React.FunctionComponent = () => {
-    return <LoginButton />;
+    return (
+        <div className="wrapper">
+            <div className="container">
+                <LoginButton />
+            </div>
+        </div>
```

Comment:
Ah I think that container is actually the wrong class for this. There is something called a section in Bulma which works way better. Also probably lets name view just so we don't overwrite properties from the container class and cause headaches later on.

This className would become `<div className="container view">`.